### ISSUE TITLE
Build for Windows 10 ARM32 

### ISSUE THREAD

**Issue by Max-RM** [telegramdesktop/tdesktop#28757]
### Is your feature request related to a problem?

There is build for Windows 10/11 ARM64, but there is no build for Windows 10 ARM32 and WinRT8(jalebroken), so can anyone compile it for this platform?

### Describe the solution you'd like

Native ARM32 exe

### Describe alternatives you've considered

.

**Comment by GeldHades27355** [comment]
You still running Windows mobile? 😁

Maybe @FrayxRulez can re-share old Arm32 Unigram for side-loading? (provided they can still connect to Telegram of course)

Modern Windows does not support 32 bit Arm apps anymore (the Arm32 subsystem was removed). I doubt that any dev will invest even a second in that architecture.

**Comment by john-preston** [comment]
@GeldHades27355 I'm not sure it's possible to build with modern tool chain even.

**Comment by Max-RM** [comment]
> You still running Windows mobile? 😁
> 
> Maybe @FrayxRulez can re-share old Arm32 Unigram for side-loading? (provided they can still connect to Telegram of course)
> 
> Modern Windows does not support 32 bit Arm apps anymore (the Arm32 subsystem was removed). I doubt that any dev will invest even a second in that architecture.

Windows 10 ARM32 15035 leaked build on Surface RT.

"Modern Windows does not support 32 bit Arm apps anymore"
I have Windows 11 ARM64 on my Xiaomi POCO x3 PRO and it still supports ARM32 apps.

**Comment by GeldHades27355** [comment]
https://www.windowscentral.com/software-apps/windows-11/windows-12-and-qualcomm-oryon-processors-will-not-support-arm32-apps

If you run insider previews, you already can't run Arm32 anymore.

You're just seeing the last gasps of Arm32.

**Comment by Max-RM** [comment]
> https://www.windowscentral.com/software-apps/windows-11/windows-12-and-qualcomm-oryon-processors-will-not-support-arm32-apps
> 
> If you run insider previews, you already can't run Arm32 anymore.
> 
> You're just seeing the last gasps of Arm32.

That's why I installed not a latest shit version of Win11ARM64.

But anyway this issue is about Windows 10 ARM32, not ARM64.

**Comment by Max-RM** [comment]
>  I doubt that any dev will invest even a second in that architecture.

Once I spent 2-3 weeks to force Python ARM32 to compile my tools to ARM32.

**Comment by Max-RM** [comment]
Idk if its helpful or not. But llvm msvc was ported to Windows ARM32: https://files.open-rt.party/Software/dev/llvm-armv7-w64-windows-msvc.zip

**Comment by GeorgeDzvengwe** [comment]
Mobile upload of 788MB Zip file on IOS 18.6.1 & save upload on to Dropbox.
File conversion & compatibility as well as accessibility on the go .

**Comment by ilya-fedin** [comment]
Qt 6.11 has dropped 32-bit support for Windows.

<img width="1371" height="684" alt="Image" src="https://github.com/user-attachments/assets/59faa9a7-535a-4b3b-add5-3d990338da53" />

**Comment by Max-RM** [comment]
> Qt 6.11 has dropped 32-bit support for Windows.
> 
> <img alt="Image" width="1371" height="684" src="https://private-user-images.githubusercontent.com/17829319/558070411-59faa9a7-535a-4b3b-add5-3d990338da53.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzI2Mjg0MzYsIm5iZiI6MTc3MjYyODEzNiwicGF0aCI6Ii8xNzgyOTMxOS81NTgwNzA0MTEtNTlmYWE5YTctNTM1YS00YjNiLWFkZDUtM2Q5OTAzMzhkYTUzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAzMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMzA0VDEyNDIxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZlNTcxM2FmODZlOTMxMGIxOTYxYjkyN2FhNzE2ZmM2ZWM3NmUwOGY0MmM1MGI5MDRiM2Y0MmIxNTVlMmM2MWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.A2b8UcA9RtW3l2NZBV3Ndpm6xgiquVG82zLVOkYqGBc">

Telegram required exactly QT 6.11 to compile?

What's the minimal QT for telegram? Because I found 6.9 ARM32 here:
https://files.open-rt.party/Software/Qt-6.9.0-winarm32.zip

**Comment by ilya-fedin** [comment]
I mean there's no sense to add architecture that is deprecated by latest Qt. There are big problems to get out of Qt 5.15 on x86/x64 and now you folks ask to add another arch to then get trapped by 6.10? I vote for this issue to be closed as not planned.

**Comment by john-preston** [comment]
This is not planned. Too much effort and I'm not sure if there will be a second user. If there is enough demand, community may build it (but there won't, I suppose).

**Comment by GeldHades27355** [comment]
Agreed. Windows dropped Arm32 also. Arm64 should be the way forward.

**Comment by Max-RM** [comment]
-_-

It would be good if someone at least compile it one last time, without future support...

**Comment by john-preston** [comment]
@Max-RM It can't be compiled one last time, because it wasn't ever compiled for ARM32 and I'm not sure it can be.
